tools/integration-test/src/tests/channel_upgrade/manual_channel_upgrade.rs

```diff
@@ -129,3 +140,129 @@ impl BinaryChannelTest for ChannelUpgradeInitHandshake {
         Ok(())
     }
 }
+
+pub struct ChannelUpgradeTryHandshake;
+
+impl TestOverrides for ChannelUpgradeTryHandshake {
+    fn modify_test_config(&self, config: &mut TestConfig) {
+        config.bootstrap_with_random_ids = false;
+    }
+
+    fn modify_relayer_config(&self, config: &mut Config) {
+        config.mode.connections.enabled = true;
+
+        config.mode.channels.enabled = false;
+        config.mode.packets.enabled = false;
+        config.mode.clients.enabled = false;
```

Comment:
As a sanity check, do we really need this since we are not spawning the supervisor at all at the moment?

Reply:
Good point, I don't think this is necessary